docs/developers-guide.md

```diff
@@ -1,8 +1,190 @@
 # Developers Guide
 
+
+
 ## Extending the GraphDB Workbench
 
+## Plugin system
+
+Since v9.1, GraphDB workbench features a plugin system which allows components to be plugged in 
```

Comment:
Kinda misleading I think. That's not the workbench version.

Reply:
I agree, we can use the workbench version here 1.2.

Reply:
:D my bad